.bazelrc

```diff
@@ -14,4 +14,6 @@ build --action_env=UI_IMAGE=quay.io/kubev2v/forklift-ui:latest
 build --action_env=UI_PLUGIN_IMAGE=quay.io/kubev2v/forklift-console-plugin:latest
 build --action_env=VALIDATION_IMAGE=quay.io/kubev2v/forklift-validation:latest
 build --action_env=VIRT_V2V_IMAGE=quay.io/kubev2v/forklift-virt-v2v:latest
+# build --action_env=OVIRT_POPULATOR_IMAGE=quay.io/kubev2v/ovirt-populator:latest
```

Comment:
this will be uncommented and the line below dropped once we post the image to `kubev2v`